### DISCUSSION TITLE
Battery level issue on RG280v

### DISCUSSION THREAD

**Discussion by Sichroteph** [eduardofilo/RG350_adam_image#79] [Hardware problems?]
My battery level starts at 70% after a charging night. (RetroArch level, same thing on the simple menu icon too that is missing a bar)

**Comment by Sichroteph** [comment]
same issue on gmenu

**Comment by Sichroteph** [comment]
Closed by error, issue still there

**Comment by eduardofilo** [comment]
Although I have not thoroughly studied everything that intervenes in the indication of the charge level, I understand that the firmware of these consoles does not have anything similar to a statistics system like modern smartphones, which is the only thing that can offer more or less consistent values. The battery is one of the most analog components in the console. I suppose that the system is limited to measuring the battery voltage and applying a function that makes that value correspond to the percentage of charge. Surely your battery offers voltage values ​​slightly lower than the usual ones in that battery model. I have a RG280V too and I can see higher values ​​of the percentage indicator as you can see in the snapshot. My advice is that you do a little study of what the values ​​you see for example in RetroArch correspond to with the real capacity of the battery by doing a complete discharge cycle and regularly logging (every half hour for example) the percentage measured. If you get a session of 2 to 4 hours of play, we can think that your battery is still operational, but that it has that "problem" whereby it supplies a voltage slightly lower than normal. In case it does not last more than 2 hours of game, change the battery and surely the problem of indicating a maximum 70% will disappear.

![screenshot004](https://user-images.githubusercontent.com/2923728/145209428-ae33b866-91a7-4d5f-8431-e1142253180f.png)

**Comment by Sichroteph** [comment]
Yes, you are probably right. The battery level doesn't seems very evolved therefore it is a rough approximation based on the voltage. Just saw that the issue still persist on a Rogue firmware, maybe I have a bad battery.
